### PR TITLE
Created basic structure LoginPage with css.

### PR DESCRIPTION
**Pull request by mateuszslowinski**

### CODE REVIEW

File: src/pages/Login.tsx [hunk 1]
```diff
@@ -1,18 +1,56 @@
...
-export default LoginPage
\ No newline at end of file
+   return (
+      <ContainerElement>
+         <LoginContainer>
+            <img
+               src="https://static1.s123-cdn-static-a.com/uploads/5191798/400_609bb5e2d9a39.png"
+               alt="logo"
+            />
+            <FormContainer onSubmit={handleClick}>
```
Comment (MS151994) [inline src/pages/Login.tsx]: Ttry # handleSubmit is more descriptive.

Comment (mateuszslowinski) [reply src/pages/Login.tsx]: done 👍

**Review by RadekJ87 — APPROVED** [review]
for me as a newbie is alright

**Review by Fremen1990 — APPROVED** [review]
Good basic structure

**Review by MS151994 — CHANGES_REQUESTED** [review]
Hold off on that PR. we need to make changes to the develop branch, then you will download the changes and adjust the component to the appearance. :)

**Pull request closed without merging**